packages/design-system-docs/src/styles/components/_UsaBanner.scss

```diff
@@ -60,10 +60,22 @@
   }
 }
 
+.c-usa-banner__guidance {
+  width: 50%;
```

Comment:
We want to put this under the small breakpoint so its not 50% on small screens.
` @media (min-width: $width-sm) {`

Reply:
Ok, i'll check that out.